src/labelle/lib/devices/dymo_labeler.py

```diff
@@ -240,23 +241,81 @@ class DymoLabeler:
 
     LABELER_DISTANCE_BETWEEN_PRINT_HEAD_AND_CUTTER_MM = 8.1
     LABELER_PRINT_HEAD_HEIGHT_MM = 8.2
-    SUPPORTED_TAPE_SIZES_MM = (24, 19, 12, 9, 6)
     DEFAULT_TAPE_SIZE_MM = 12
 
+    # Default supported tape sizes
+    labeler_supported_tape_sizes: list
+
+    # Default printer head size in dots
+    labeler_print_head_width_pixels: int
+
+    # Default printer margins (for each tape size)
+    # { tapeSizeMm : (firstVisiblePixel, lastVisiblePixel) }
+    labeler_label_vertical_margins: dict[int, Any]
```

Comment:
Maybe something like:
```
labeler_label_vertical_margins: dict[int, tuple[int, int]]
```

Reply:
I am wondering what "first" and "last" mean here regarding visible pixels. I don't know offhand which pixel ordering is being used, so it would be helpful to be more explicit.

Reply:
@tomers Thanks, will try again. Had some linter issues with this...

@maresb Me neither (at the time) Now I know it is "first visible pixel from the top" I will extend the comment.

Reply:
This is outdated btw